### ISSUE TITLE
flatpak apps issue

### ISSUE THREAD

**Issue by dikasetyaprayogi** [flexagoon/rounded-window-corners#43]
thanks for continuing this great extension, just want to made some report that flatpak apps tend to have some issue in overview and "+ Add window" function in blacklist and custom setting also doesn't work

os: Fedora Silverblue 40, gnome 46, extension version 12

![Screenshot from 2024-09-15 09-13-52](https://github.com/user-attachments/assets/8bc21dc7-d2cc-4b08-9013-c47a19f10fb3)

**Comment by flexagoon** [comment]
> extension version 12

Are you sure?.. the latest available version of this extension is 3

**Comment by Sakugits** [comment]
It also happens to me on Vesktop, I don´t know if this happens on other apps.

![imagen](https://github.com/user-attachments/assets/79b1c026-2797-4bfc-8981-e5ae76a1256e)

**Comment by dikasetyaprayogi** [comment]
@flexagoon

> Are you sure?.. The latest available version of this extension is 3.

Yes I'm using the latest version 3, I see it 12 from extension manager app and it actually means the revision number from extension gnome org (EGO) site, not the ones in metadata.

> How did you install the extension?

Tried from EGO site and via extension manager app - same result.

I mean by extension manager is this https://flathub.org/apps/com.mattjakeman.ExtensionManager

**Comment by Plyply99** [comment]
This is a flatpak permissions issue concerning wayland / xwayland. Use flatseal to set permissions.
![2024-09-24-060035_hyprshot](https://github.com/user-attachments/assets/68bed5f1-f8dc-43f0-ae91-db37ecd8ac83)

**Comment by dikasetyaprayogi** [comment]
using flatseal and the issue still persist

![Screenshot from 2024-09-26 07-45-39](https://github.com/user-attachments/assets/d53b8104-f5ad-4969-950d-44a52adc779d)

**Comment by zuusuuz** [comment]
Also have this issue but only with Vesktop/Armcord. Currently have 45 Flatpaks installed and all but Vesktop work fine in Arch on a 7900XTX so I assume some sort of specific Electron bug.
In Vesktop the only workaround I've found is to enable the Discord Titlebar. You'll lose the nice Adwaita buttons but the big borders are gone.
![Screenshot From 2024-11-02 10-11-14](https://github.com/user-attachments/assets/dc29b24e-423b-4975-bb1d-af931ad4c9d1)

**Comment by flexagoon** [comment]
@dikasetyaprayogi can you check if this still happens?
